Jellyfin.Data/Entities/Security/DeviceOptions.cs

```diff
@@ -0,0 +1,35 @@
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Jellyfin.Data.Entities.Security
+{
+    /// <summary>
+    /// An entity representing custom options for a device.
+    /// </summary>
+    public class DeviceOptions
```

Comment:
Why this can't go with Device?

Reply:
Device might not be the best name, but it is a session for a device. DeviceOptions are for properties that are device-wide, and not specific to a session.